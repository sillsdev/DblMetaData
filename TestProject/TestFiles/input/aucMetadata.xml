<?xml version='1.0' encoding='utf-8'?>
<DBLMetadata id="7203006e38b66a06" revision="1" type="text" typeVersion="1.2">
  <identification>
    <name>Wængonguï nänö apæ̈negaïnö ante näni yewæ̈mongainta</name>
    <nameLocal>Wængonguï nänö apæ̈negaïnö ante näni yewæ̈mongainta</nameLocal>
    <abbreviation>aucNT</abbreviation>
    <abbreviationLocal>aucNT</abbreviationLocal>
    <scope>WNT:New Testament</scope>
    <description>1st ed.</description>
    <dateCompleted>2009</dateCompleted>
    <systemId type="reap">http://www.reap.insitehome.org/handle/9284745/10171</systemId>
    <systemId type="paratext">7203006e38b66a0677b116b141b2b5fcba07c44a</systemId>
    <bundleProducer>Paratext/7.3.0.60</bundleProducer>
  </identification>
  <confidential>false</confidential>
  <agencies>
    <etenPartner>WBT</etenPartner>
    <creator>Wycliffe Inc.</creator>
    <publisher>Wycliffe Inc.</publisher>
  </agencies>
  <language>
    <iso>auc</iso>
    <name>Waorani</name>
    <script>Latin</script>
    <scriptDirection>LTR</scriptDirection>
    <ldml />
    <rod />
    <numerals />
  </language>
  <country>
    <iso>EC</iso>
    <name>Ecuador</name>
  </country>
  <type>
    <translationType>New</translationType>
    <audience>Common</audience>
  </type>
  <bookNames>
    <book code="MAT">
      <long>Itota Codito ingantedö ante Mäateo nänö watapæ̈ ante yewæ̈mongainta</long>
      <short>Mäateo</short>
      <abbr>Mäateo</abbr>
    </book>
    <book code="MRK">
      <long>Itota Codito ingantedö ante Mäadoco nänö watapæ̈ ante yewæ̈mongainta</long>
      <short>Mäadoco</short>
      <abbr>Mäadoco</abbr>
    </book>
    <book code="LUK">
      <long>Itota Codito ingantedö ante Odoca nänö watapæ̈ ante yewæ̈mongainta</long>
      <short>Odoca</short>
      <abbr>Odoca</abbr>
    </book>
    <book code="JHN">
      <long>Itota Codito ingantedö ante Wäö nänö watapæ̈ ante yewæ̈mongainta</long>
      <short>Wäö</short>
      <abbr>Wäö</abbr>
    </book>
    <book code="ACT">
      <long>Itota nänö da godongaïnäni Näni Cægaïnö</long>
      <short>Näni Cægaïnö</short>
      <abbr>Näni Cægaïnö</abbr>
    </book>
    <book code="ROM">
      <long>Odömänoidi ïnänite Pabodo nänö cadota ante yewæ̈mongainta</long>
      <short>Odömänoidi</short>
      <abbr>Odömänoidi</abbr>
    </book>
    <book code="1CO">
      <long>1 Coodintoidi ïnänite Pabodo cadota ante tänota nänö yewæ̈mongainta</long>
      <short>1 Coodintoidi</short>
      <abbr>1 Coodintoidi</abbr>
    </book>
    <book code="2CO">
      <long>2 Coodintoidi ïnänite Pabodo cadota ante ayænta nänö</long>
      <short>2 Coodintoidi</short>
      <abbr>2 Coodintoidi</abbr>
    </book>
    <book code="GAL">
      <long>Gadataidi ïnänite Pabodo nänö cadota ante yewæ̈mongainta</long>
      <short>Gadataidi</short>
      <abbr>Gadataidi</abbr>
    </book>
    <book code="EPH">
      <long>Epetoidi ïnänite Pabodo nänö cadota ante yewæ̈mongainta</long>
      <short>Epetoidi</short>
      <abbr>Epetoidi</abbr>
    </book>
    <book code="PHP">
      <long>Pidipenteidi ïnänite Pabodo nänö cadota ante yewæ̈mongainta</long>
      <short>Pidipenteidi</short>
      <abbr>Pidipenteidi</abbr>
    </book>
    <book code="COL">
      <long>Codotenteidi ïnänite Pabodo nänö cadota ante yewæ̈mongainta</long>
      <short>Codotenteidi</short>
      <abbr>Codotenteidi</abbr>
    </book>
    <book code="1TH">
      <long>1 Tetadönitenteidi ïnänite Pabodo cadota ante tänota nänö yewæ̈mongainta</long>
      <short>1 Tetadönitenteidi</short>
      <abbr>1 Tetadönitenteidi</abbr>
    </book>
    <book code="2TH">
      <long>2 Tetadönitenteidi ïnänite Pabodo cadota ante ayænta nänö yewæ̈mongainta</long>
      <short>2 Tetadönitenteidi</short>
      <abbr>2 Tetadönitenteidi</abbr>
    </book>
    <book code="1TI">
      <long>1 Tïmoteo ingante Pabodo cadota ante tänota nänö yewæ̈mongainta</long>
      <short>1 Tïmoteo</short>
      <abbr>1 Tïmoteo</abbr>
    </book>
    <book code="2TI">
      <long>2 Tïmoteo ingante Pabodo cadota ante ayænta nänö yewæ̈mongainta</long>
      <short>2 Tïmoteo</short>
      <abbr>2 Tïmoteo</abbr>
    </book>
    <book code="TIT">
      <long>Tito ingante Pabodo cadota ante nänö yewæ̈mongainta</long>
      <short>Tito</short>
      <abbr>Tito</abbr>
    </book>
    <book code="PHM">
      <long>Pidemöö ingante Pabodo cadota ante nänö yewæ̈mongainta</long>
      <short>Pidemöö</short>
      <abbr>Pidemöö</abbr>
    </book>
    <book code="HEB">
      <long>Ebedeoidi ïnänite cadota ante näni yewæ̈mongainta</long>
      <short>Ebedeoidi</short>
      <abbr>Ebedeoidi</abbr>
    </book>
    <book code="JAS">
      <long>Tantiago cadota ante nänö yewæ̈mongainta</long>
      <short>Tantiago</short>
      <abbr>Tantiago</abbr>
    </book>
    <book code="1PE">
      <long>1 Pegodo cadota ante nänö tänota yewæ̈mongainta</long>
      <short>1 Pegodo</short>
      <abbr>1 Pegodo</abbr>
    </book>
    <book code="2PE">
      <long>2 Pegodo cadota ante nänö ayænta yewæ̈mongainta</long>
      <short>2 Pegodo</short>
      <abbr>2 Pegodo</abbr>
    </book>
    <book code="1JN">
      <long>1 Wäö cadota ante nänö tänota yewæ̈mongainta</long>
      <short>1 Wäö</short>
      <abbr>1 Wäö</abbr>
    </book>
    <book code="2JN">
      <long>2 Wäö cadota ante nänö ayænta yewæ̈mongainta</long>
      <short>2 Wäö</short>
      <abbr>2 Wäö</abbr>
    </book>
    <book code="3JN">
      <long>3 Wäö cadota ante nänö tömanta yewæ̈mongainta</long>
      <short>3 Wäö</short>
      <abbr>3 Wäö</abbr>
    </book>
    <book code="JUD">
      <long>Codaa cadota ante nänö yewæ̈mongainta</long>
      <short>Codaa</short>
      <abbr>Codaa</abbr>
    </book>
    <book code="REV">
      <long>Wäö ingante Itota Codito Nänö Odömongaïnö</long>
      <short>Nänö Odömongaïnö</short>
      <abbr>Nänö Odömongaïnö</abbr>
    </book>
  </bookNames>
  <contents>
    <bookList id="default">
      <name>Wængonguï nänö apæ̈negaïnö ante näni yewæ̈mongainta</name>
      <nameLocal>Wængonguï nänö apæ̈negaïnö ante näni yewæ̈mongainta</nameLocal>
      <abbreviation>aucNT</abbreviation>
      <abbreviationLocal>aucNT</abbreviationLocal>
      <description>NT:Segunda edición</description>
      <range>Protestant New Testament (27 books)</range>
      <tradition>Western Protestant order</tradition>
      <division id="NT">
        <books>
          <book code="MAT" />
          <book code="MRK" />
          <book code="LUK" />
          <book code="JHN" />
          <book code="ACT" />
          <book code="ROM" />
          <book code="1CO" />
          <book code="2CO" />
          <book code="GAL" />
          <book code="EPH" />
          <book code="PHP" />
          <book code="COL" />
          <book code="1TH" />
          <book code="2TH" />
          <book code="1TI" />
          <book code="2TI" />
          <book code="TIT" />
          <book code="PHM" />
          <book code="HEB" />
          <book code="JAS" />
          <book code="1PE" />
          <book code="2PE" />
          <book code="1JN" />
          <book code="2JN" />
          <book code="3JN" />
          <book code="JUD" />
          <book code="REV" />
        </books>
      </division>
    </bookList>
  </contents>
  <progress>
    
  </progress>
  <contact>
    <rightsHolder>Wycliffe Inc.</rightsHolder>
    <rightsHolderLocal>Wycliffe Inc.</rightsHolderLocal>
    <rightsHolderAbbreviation>WBT</rightsHolderAbbreviation>
    <rightsHolderURL>http://www.wycliffe.org</rightsHolderURL>
    <rightsHolderFacebook>http://www.facebook.com/WycliffeUSA</rightsHolderFacebook>
  </contact>
  <copyright>
    <statement contentType="xhtml">© 2009, Wycliffe Inc. All rights reserved.</statement>
  </copyright>
  <promotion>
    <promoVersionInfo contentType="xhtml">
      <p>Waorani (auc) Ecuador<br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
      </p>
      <h2>Copyright Information</h2>
      <p>© 2009, Wycliffe Inc. All rights reserved.</p>
      <p>This translation text is made available to you under the
terms of the Creative Commons License: Attribution-Noncommercial-No Derivative Works.
(<a href="http://creativecommons.org/licenses/by-nc-nd/3.0/">http://creativecommons.org/licenses/by-nc-nd/3.0/</a>
      </p>
      <p>You may share, copy, distribute, transmit, and extract portions
or quotations from this work, provided that you include the above copyright
information:</p>
      <ul>
        <li>You must give Attribution to the work</li>
        <li>You do not sell this work for a profit.</li>
        <li>You do not make any derivative
works that change any of the actual words or punctuation of the Scriptures.</li>
      </ul>
      <p>Permissions beyond the scope of this license may be
available if you <a href="mailto:ScriptureCopyrightPermission_Intl@sil.org">contact
us</a>
      </p>
      <p>NT:Segunda edición<br />
        <b>The New Testament</b>
        <br />
        <a href="http://www.ethnologue.com/show_language.asp?code=auc">Waorani</a>
      </p>
    </promoVersionInfo>
    <promoEmail contentType="xhtml">
      <p>Waorani (auc) Ecuador<br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
      </p>
      <h2>Copyright Information</h2>
      <p>© 2009, Wycliffe Inc. All rights reserved.</p>
      <p>This translation text is made available to you under the
terms of the Creative Commons License: Attribution-Noncommercial-No Derivative Works.
(<a href="http://creativecommons.org/licenses/by-nc-nd/3.0/">http://creativecommons.org/licenses/by-nc-nd/3.0/</a>
      </p>
      <p>You may share, copy, distribute, transmit, and extract portions
or quotations from this work, provided that you include the above copyright
information:</p>
      <ul>
        <li>You must give Attribution to the work</li>
        <li>You do not sell this work for a profit.</li>
        <li>You do not make any derivative
works that change any of the actual words or punctuation of the Scriptures.</li>
      </ul>
      <p>Permissions beyond the scope of this license may be
available if you <a href="mailto:ScriptureCopyrightPermission_Intl@sil.org">contact
us</a>
      </p>
      <p>NT:Segunda edición<br />
        <b>The New Testament</b>
        <br />
        <a href="http://www.ethnologue.com/show_language.asp?code=auc">Waorani</a>
      </p>
    </promoEmail>
  </promotion>
  <archiveStatus>
    <archivistName>Lois Gourley</archivistName>
    <dateArchived>2011-11-18T01:01:01</dateArchived>
    <dateUpdated>2011-11-21T01:01:01</dateUpdated>
    <comments>n/a</comments>
  </archiveStatus>
  <format>text/xml</format>
</DBLMetadata>
