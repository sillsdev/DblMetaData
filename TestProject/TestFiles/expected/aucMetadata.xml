<?xml version="1.0" encoding="utf-8"?><?xml-model href="metadataWbt-1.2.rnc" type="application/relax-ng-compact-syntax"?><DBLMetadata id="2880c78491b2f8ce" revision="3" type="text" typeVersion="1.2" xp_0:base="http://purl.org/ubs/metadata/dc/terms/" xmlns:xp_0="http://www.w3.org/XML/1998/namespace"><identification><name xp_1:propertyURI="title" xmlns:xp_1="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">Wængonguï nänö apæ̈negaïnö ante näni yewæ̈mongainta</name><nameLocal>Wængonguï nänö apæ̈negaïnö ante näni yewæ̈mongainta</nameLocal><abbreviation>aucNT</abbreviation><abbreviationLocal>aucNT</abbreviationLocal><scope xp_2:propertyURI="title/scriptureScope" xmlns:xp_2="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">NT</scope><description xp_3:propertyURI="description" xmlns:xp_3="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">New Testament in Waorani</description><dateCompleted xp_4:propertyURI="date" xp_4:sesURI="http://purl.org/dc/terms/W3CDTF" xmlns:xp_4="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">2009</dateCompleted><systemId type="reap" xp_5:propertyURI="identifier/reapID" xmlns:xp_5="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">http://www.reap.insitehome.org/handle/9284745/10171</systemId><systemId type="paratext" xp_6:propertyURI="identifier/ptxID" xmlns:xp_6="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/"></systemId><systemId type="tms" xp_7:propertyURI="identifier/tmsID" xmlns:xp_7="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">seeReap</systemId><bundleProducer>Paratext/7.3.0.60</bundleProducer></identification><confidential xp_8:propertyURI="accessRights/confidential" xmlns:xp_8="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">false</confidential><agencies><etenPartner>WBT</etenPartner><creator xp_9:propertyURI="creator" xmlns:xp_9="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">Wycliffe Inc.</creator><publisher xp_10:propertyURI="publisher" xmlns:xp_10="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">Wycliffe Inc.</publisher><contributor xp_11:propertyURI="contributor" xmlns:xp_11="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/"></contributor></agencies><language><iso xp_12:propertyURI="language/iso" xp_12:sesURI="http://purl.org/dc/terms/ISO639-3" xmlns:xp_12="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">auc</iso><name xp_13:propertyURI="subject/subjectLanguage" xp_13:sesURI="http://purl.org/dc/terms/ISO639-3" xmlns:xp_13="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">Waorani</name><ldml xp_14:propertyURI="language/ldml" xmlns:xp_14="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">en</ldml><rod xp_15:propertyURI="language/rod" xmlns:xp_15="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/"></rod><script xp_16:propertyURI="language/script" xmlns:xp_16="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">Latin</script><scriptDirection xp_17:propertyURI="language/scriptDirection" xmlns:xp_17="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">LTR</scriptDirection><numerals xp_18:propertyURI="language/numerals" xmlns:xp_18="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/"></numerals></language><country><iso xp_19:propertyURI="coverage/spatial" xp_19:sesURI="http://purl.org/dc/terms/ISO3166" xmlns:xp_19="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">EC</iso><name xp_20:propertyURI="subject/subjectCountry" xmlns:xp_20="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">Ecuador</name></country><type><translationType xp_21:propertyURI="type/translationType" xmlns:xp_21="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">New</translationType><audience xp_22:propertyURI="audience" xmlns:xp_22="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">Common</audience></type><bookNames /><contents xp_23:propertyURI="tableOfContents" xmlns:xp_23="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/"><bookList id="default"><name>Wængonguï nänö apæ̈negaïnö ante näni yewæ̈mongainta</name><nameLocal>Wængonguï nänö apæ̈negaïnö ante näni yewæ̈mongainta</nameLocal><abbreviation>aucNT</abbreviation><abbreviationLocal>aucNT</abbreviationLocal><description>NT</description><range>Protestant New Testament (27 books)</range><tradition>Western Protestant order</tradition><division id="NT"><books><book code="MAT"></book><book code="MRK"></book><book code="LUK"></book><book code="JHN"></book><book code="ACT"></book><book code="ROM"></book><book code="1CO"></book><book code="2CO"></book><book code="GAL"></book><book code="EPH"></book><book code="PHP"></book><book code="COL"></book><book code="1TH"></book><book code="2TH"></book><book code="1TI"></book><book code="2TI"></book><book code="TIT"></book><book code="PHM"></book><book code="HEB"></book><book code="JAS"></book><book code="1PE"></book><book code="2PE"></book><book code="1JN"></book><book code="2JN"></book><book code="3JN"></book><book code="JUD"></book><book code="REV"></book></books></division></bookList></contents><contact><rightsHolder xp_24:propertyURI="rightsHolder" xmlns:xp_24="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">Wycliffe Inc.</rightsHolder><rightsHolderLocal xp_25:propertyURI="rightsHolder/contactLocal" xmlns:xp_25="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">Wycliffe Inc.</rightsHolderLocal><rightsHolderAbbreviation xp_26:propertyURI="rightsHolder/contactAbbreviation" xmlns:xp_26="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">WBT</rightsHolderAbbreviation><rightsHolderURL xp_27:propertyURI="rightsHolder/contactURL" xmlns:xp_27="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">http://www.wycliffe.org</rightsHolderURL><rightsHolderFacebook xp_28:propertyURI="rightsHolder/contactFacebook" xmlns:xp_28="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">http://www.facebook.com/WycliffeUSA</rightsHolderFacebook></contact><copyright><statement contentType="xhtml" xp_29:propertyURI="rights" xmlns:xp_29="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">© 2009, Wycliffe Inc. All rights reserved.</statement></copyright><promotion><promoVersionInfo contentType="xhtml" xp_30:propertyURI="description/pubPromoVersionInfo" xmlns:xp_30="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/"><p>Waorani (auc) Ecuador<br /></p><h2>Copyright Information</h2><p>© 2009, Wycliffe Inc. All rights reserved.</p><p>This translation text is made available to you under the
terms of the Creative Commons License: Attribution-Noncommercial-No Derivative Works.
(<a href="http://creativecommons.org/licenses/by-nc-nd/3.0/">http://creativecommons.org/licenses/by-nc-nd/3.0/</a></p><p>You may share, copy, distribute, transmit, and extract portions
or quotations from this work, provided that you include the above copyright
information:</p><ul><li>You must give Attribution to the work</li><li>You do not sell this work for a profit.</li><li>You do not make any derivative
works that change any of the actual words or punctuation of the Scriptures.</li></ul><p>Permissions beyond the scope of this license may be
available if you <a href="mailto:ScriptureCopyrightPermission_Intl@sil.org">contact
us</a></p><p>NT:Segunda edición<br /><b>The New Testament</b><br /><a href="http://www.ethnologue.com/show_language.asp?code=auc">Waorani</a></p></promoVersionInfo><promoEmail contentType="xhtml" xp_31:propertyURI="description/pubPromoEmail" xmlns:xp_31="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/"><p>Hi YouVersion friend,</p><p>Nice work downloading the <em>Wængonguï nänö apæ̈negaïnö ante näni yewæ̈mongainta</em> (New Testament in Waorani) in the Bible App! Now you'll have anytime, anywhere access to God's Word on your mobile device—even if you're outside of service coverage or not connected to the Internet. It also means faster service whenever you read that version since it's stored on your device. Enjoy!</p><p>This download was made possible by Wycliffe Inc. We really appreciate their passion for making the Bible available to millions of people around the world. Because of their generosity, YouVersion users like you can open up the Bible and hear from God no matter where you are. You can learn more about the great things Wycliffe Inc. is doing on many fronts by visiting <a href="http://www.wycliffe.org">www.wycliffe.org.</a></p><p>Again, we're glad you downloaded the <em>Wængonguï nänö apæ̈negaïnö ante näni yewæ̈mongainta</em> (New Testament in Waorani) and hope it enriches your interaction with God's Word.</p><p>Your Friends at YouVersion</p></promoEmail></promotion><archiveStatus><archivistName xp_32:propertyURI="contributor/archivist" xmlns:xp_32="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">Lois Gourley</archivistName><dateArchived xp_33:propertyURI="dateSubmitted" xp_33:sesURI="http://purl.org/dc/terms/W3CDTF" xmlns:xp_33="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">2011-11-18T01:01:01</dateArchived><dateUpdated xp_34:propertyURI="modified" xp_34:sesURI="http://purl.org/dc/terms/W3CDTF" xmlns:xp_34="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">2011-11-21T01:01:01</dateUpdated><comments xp_35:propertyURI="abstract" xmlns:xp_35="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">n/a</comments></archiveStatus><format xp_36:propertyURI="format" xp_36:sesURI="http://purl.org/dc/terms/IMT" xmlns:xp_36="http://purl.org/dc/xmlns/2008/09/01/dc-ds-xml/">text/xml</format></DBLMetadata>